### PR TITLE
Add the style guide route and components

### PR DESCRIPTION
**Pull request by mkitt**
This is so we can better tweak the work coming up with the icons. Also
adds a bit of the cso/jso styling.

[#140987175](https://www.pivotaltracker.com/story/show/140987175)

### CODE REVIEW

**Comment by ellodanger** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-kind="Warning">
          1 Warning
      </th>
     </tr>
  </thead>
  <tbody>
    <tr>
      <td>:warning:</td>
      <td data-sticky="false">Big PR</td>
    </tr>
  </tbody>
</table>
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-kind="Message">
          1 Message
      </th>
     </tr>
  </thead>
  <tbody>
    <tr>
      <td>:book:</td>
      <td data-sticky="false">Nice, more deletions than insertions :red_circle:</td>
    </tr>
  </tbody>
</table>

<p align="right" data-meta="generated_by_danger">
  Generated by :no_entry_sign: <a href="http://danger.systems/">danger</a>
</p>

File: src/routes/styleguide/index.js [hunk 1]
```diff
@@ -0,0 +1,11 @@
+import StyleGuide from '../../components/devtools/StyleGuide'
+
+export default [
+  {
+    path: 'styleguide',
+    getComponents(location, cb) {
```
Comment (mkitt) [inline src/routes/styleguide/index.js]: We would be hijacking this from the user which is cool with me. ✈️

**Comment by mkitt** [comment]
@rynbyjn 0ebdcac has the `ENV` flag in it. Added a couple of more commits after it too. Another pair of 👀 wouldn't hurt on this one. ✌️

File: src/containers/NavbarContainer.js [hunk 2]
```diff
@@ -84,6 +84,7 @@ class NavbarContainer extends PureComponent {
   }
 
   static defaultProps = {
+    activeTabType: '',
```
Comment (rynbyjn) [inline src/containers/NavbarContainer.js]: Still weird to me that this isn't getting a default value from the gui reducer..

Comment (mkitt) [inline src/containers/NavbarContainer.js]: I know.. the warning was making me crazy. 🎱

**Pull request merged**